### ISSUE TITLE
Support array key navigation in Tabs

### ISSUE THREAD

**Issue by argyleink** [argyleink/gui-challenges#112]
focus on the nav of links and if the arrows are used, shift between the tabs.
could also add roving-ux so state is saved?

shoutout to Rob Levin for the [idea](https://twitter.com/roblevintennis/status/1516597976973647872?t=lYwMgxdCsgCbnVTOt_KT1w&s=03)

**Comment by roblevintennis** [comment]
Nice @argyleink -- I also noticed on the modal demo tabs aren't trapped. Not sure if that's something else you're interested in. I have too much work on the AgnosticUI stuff at the moment so unfortunately can't lend a helping hand :(

**Comment by argyleink** [comment]
> Nice @argyleink -- I also noticed on the modal demo tabs aren't trapped. Not sure if that's something else you're interested in. I have too much work on the AgnosticUI stuff at the moment so unfortunately can't lend a helping hand :(

yeah, no focus trapping, just inert, like how `window.alert()` works. that'd be a nice remix idea tho! sharing "your style" and the things you find important 🙂

**Comment by roblevintennis** [comment]
I'm implemented svelte-a11y-dialog and angular-a11y-dialog and ported Vue-a11y-dialog to Vue 3 (and then delegated to those from AgnosticUI) but those don't use native dialog for the issues documented by `a11y-dialog` 🤓🙂

**Comment by shivam200446** [comment]
hey, I want to work on it. assign me!

**Comment by argyleink** [comment]
> hey, I want to work on it. assign me!

sweet! you got it!
